src/settings/FeeFinesTable/FeeFinesTable.css

```diff
@@ -0,0 +1,17 @@
+@import "@folio/stripes-components/lib/variables.css";
+
+.customCol {
```

Comment:
Is this being used anywhere?

Reply:
Yup, it's been used in ChargeNotice.js file in the current folder